crates/common/fmt/src/dynamic.rs

```diff
@@ -36,7 +36,7 @@ impl DynValueFormatter {
                 f.write_str("]")
             }
             DynSolValue::Tuple(values) => self.tuple(values, f),
-            DynSolValue::String(inner) => write!(f, "{inner:?}"), // escape strings
+            DynSolValue::String(inner) => write!(f, "{}", inner.escape_debug()), // escape strings
```

Comment:
Hmm I think we want the quotes when not raw, so that they're displayed in traces
Your initial implementation was right, just missing the escape debug in the first branch
Sorry about this, could you revert the last changes and add the escape_debug in the raw branch?

Reply:
No worries! Should be good for another look now 😄